embeddings/pipeline/lightning_sequence_labeling.py

```diff
@@ -87,9 +87,11 @@ def __init__(
             load_dataset_kwargs=load_dataset_kwargs,
             **self.datamodule_kwargs
         )
+        datamodule.setup()
```

Comment:
i do not like calling it manually. According to the warning it will be deprecated in the new version.

`pytorch_lightning/core/datamodule.py:469: LightningDeprecationWarning: DataModule.setup has already been called, so it will not be called again. In v1.6 this behavior will change to always call DataModule.setup.`

Reply:
We can either call it manually or ask user to give the num_classes argument to the pipelines since it is needed to initialize the lightning module

Reply:
Same as above